pkg/sloop/webserver/webfiles_test.go

```diff
@@ -0,0 +1,48 @@
+package webserver
+
+import (
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path"
+	"testing"
+)
+
+const (
+	defaultFileMode = os.FileMode(0755)
+	someContents1   = "contents abcd"
+	filePath        = "webfiles/index.html"
+)
+
+func Test_BindataReadWebfile_True(t *testing.T) {
+	expectedOutput, err := Asset(filePath)
+	assert.Nil(t, err)
+
+	actualOutput, _ := readWebfile(filePath, nil, &afero.Afero{afero.NewMemMapFs()})
+	assert.Equal(t, actualOutput, expectedOutput)
```

Comment:
I think the `assert.Equal` parameters have expected first, then actual. If this is correct, please align yours to match.